28-Github Profile/script.js

```diff
@@ -0,0 +1,101 @@
+const ApiUrl = 'https://api.github.com/users/'
+const form = document.getElementById('form')
+const search = document.getElementById('search')
+const main = document.getElementById('main')
+
+
+async function getUser(username) {
+  try {
+    search.disabled = true
+    const { data } = await axios(ApiUrl + username)
+
+    createUserCard(data)
+    getRepos(username)
```

Comment:
I think there must be a mistake here, `getRepos` is an `async` function and must be `await`ed for, no?